mlflow/sagemaker/__init__.py

```diff
@@ -163,6 +164,7 @@ def push_image_to_ecr(image=DEFAULT_IMAGE_NAME):
 def deploy(
     app_name,
     model_uri,
+    variant_name=None,
```

Comment:
Nit: For backwards compatibility, can we add `variant_name` to the end of the argument list?

Reply:
Can do.